.github/workflows/ci.yml

```diff
@@ -15,91 +15,89 @@ jobs:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
         python-version: ["3.7", "3.8", "3.9", "3.10", "3.11"]
-        parser: [pure, native]
         exclude:
           # skip these builds until https://bugs.launchpad.net/lxml/+bug/1977998 is resolved
           - os: windows-latest
             python-version: "3.11"
     steps:
       - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
```

Comment:
Do we need this for these jobs?

Reply:
Without them, hatch fails to create the environment because setuptools_scm shows the version as `0.1+gitsomething`, but the dev tools like usort depend on `libcst>=0.3.7`, creating a version mismatch. Not sure if there's a better fix, but this at least makes it work, and the repo isn't so big that I feel guilty doing it 😅